anvil-python/anvil/jobs/checks.py

```diff
@@ -43,3 +42,38 @@ def run(self) -> bool:
             LOG.warning(self.message)
 
         return True
+
+
+class DaemonGroupCheck(SunbeamDaemonGroupCheck):
+    """Check if user is member of socket group."""
+
+    def run(self) -> bool:
+        if not os.access(self.clusterd_socket, os.W_OK):
+            self.message = (
+                "Insufficient permissions to run maas-anvil commands\n"
+                f"Add the user {self.user!r} to the {self.group!r} group:\n"
+                "\n"
+                f"    sudo usermod -a -G {self.group} {self.user}\n"
+                "\n"
+                "After this, reload the user groups either via a reboot or by"
+                f" running 'newgrp {self.group}'."
+            )
```

Comment:
Is it worth converting this to a multiline string to cleanup the `\n` and `"` that are all over this?

Reply:
Since the only purpose of those checks was to replace `sunbeam` references from the failure messages, I updated the PR to do exactly that. Could you please re-review?